### PR TITLE
Added priority parameter

### PR DESCRIPTION
**Pull request by agronick**
Hey, I added the ability to set an optional priority parameter on each job in the callback client.

### CODE REVIEW

File: aiogear/callback_client.py [hunk 1]
```diff
@@ -72,7 +74,15 @@ def set_update_callback(self, async_callback):
             'total': len(jobs)
         })
 
-        async def submit_each(uuid, data):
```
Comment (sardok) [inline aiogear/callback_client.py:75]: I think its better to have it as parameter with default.
```
async def submit_each(uuid, data, priority=PACKET_TYPES.SUBMIT_JOB):
 ...
```

File: aiogear/callback_client.py [hunk 2]
```diff
@@ -82,8 +92,15 @@ def set_update_callback(self, async_callback):
             Completed future contains a generator of job results
             """
```
Comment (sardok) [inline aiogear/callback_client.py:94]: Following if/elif/else checks could be replace by more idiomatic code like,

```
priority_map = {
    PACKET_TYPES.SUBMIT_JOB_HIGH: self.submit_job_high
    PACKET_TYPES.SUBMIT_JOB_LOW: self.submit_job_low
    PACKET_TYPES.SUBMIT_JOB: self.submit_job
}

submit_funct = priority_map.get(priority)
if not submit_funct:
    raise Exception("Unsupported priority {}".format(priority))
```

**Review by sardok — CHANGES_REQUESTED** [review]

**Comment by sardok** [comment]
@agronick sorry for late reply Kyle, i've just had a time to review the PR.

**Comment by agronick** [comment]
Thanks Ill make the changes tomorrow.

**Comment by sardok** [comment]
@agronick Created PR6 with additional fixes.

**Pull request closed without merging**